index.js

```diff
@@ -27,16 +26,11 @@ class Client {
   constructor (auth, inputApiUrl = defaultApiUrl) {
     const { ethAddress, apiKey, password } = auth || {}
 
-    let userId
-    if (!password && !ethAddress && !apiKey) {
-      userId = trialUserId
+    if (!apiKey && !password && !ethAddress) {
+      throw new TypeError('Please provide auth options.')
     }
 
-    if (password && !ethAddress && !apiKey) {
-      throw new TypeError('Please provide an user id auth option.')
-    }
-
-    if (!apiKey && !userId && (!password && ethAddress)) {
+    if (!apiKey && (!password || !ethAddress)) {
       throw new TypeError('Please provide a password auth option.')
```

Comment:
With these changes i think the error message is wrong now, maybe `Please provide either ethAddress and password or apiKey`, this is actually what we are checking now, WDYT?

Reply:
Sounds good to me.

Reply:
Done!